### DISCUSSION TITLE
No flows are shown

### DISCUSSION THREAD

**Discussion by khusleek** [akvorado/akvorado#1462] [Q&A] [answered]
Hello, 
I recently installed Akvorado using Docker Compose, and all containers are up and running. I've configured my Cisco router with IP 192.168.6.90 to send NetFlow data to the Akvorado. However, when I access the Akvorado web interface, I’m not seeing any flows or exporters displayed.

![akvorado web](https://github.com/user-attachments/assets/8fbdb1ca-43bb-493a-a73a-3b859feb8c35)
Despite all containers running and the below metrics indicating that data is being received, no flows or exporters appear in the web UI. Is there anything I might be missing, or specific cache or configuration settings in Akvorado that I should adjust? Any guidance would be appreciated!

Thank you!
### Outputs from troubleshooting commands:

- curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep core

akvorado_inlet_core_classifier_exporter_cache_size_items 0
akvorado_inlet_core_classifier_interface_cache_size_items 0
akvorado_inlet_core_flows_http_clients 0

- curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep decoder

akvorado_inlet_flow_decoder_flows_total{name="netflow"} 2045
akvorado_inlet_flow_decoder_netflow_flows_total{exporter="192.168.6.90",version="9"} 2045
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="192.168.6.90",type="OptionsDataFlowSet",version="9"} 2012
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="192.168.6.90",type="OptionsTemplateFlowSet",version="9"} 2012
akvorado_inlet_flow_decoder_netflow_templates_total{exporter="192.168.6.90",obs_domain_id="6",template_id="258",type="options_template",version="9"} 2012

- curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet_flow_input_udp_packets'

akvorado_inlet_flow_input_udp_packets_total{exporter="192.168.6.90",listener=":2055",worker="3"} 2083

- curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep kafka

akvorado_inlet_kafka_brokers_incoming_byte_rate{broker="1001"} 0.01252574117016354
akvorado_inlet_kafka_brokers_inflight_requests{broker="1001"} 0
akvorado_inlet_kafka_brokers_outgoing_byte_rate{broker="1001"} 0.00026953178847931124
akvorado_inlet_kafka_brokers_request_latency_ms_bucket{broker="1001",le="0.5"} 0
akvorado_inlet_kafka_brokers_request_latency_ms_bucket{broker="1001",le="0.9"} 0
akvorado_inlet_kafka_brokers_request_latency_ms_bucket{broker="1001",le="0.99"} 2
akvorado_inlet_kafka_brokers_request_latency_ms_bucket{broker="1001",le="+Inf"} 34
akvorado_inlet_kafka_brokers_request_latency_ms_sum{broker="1001"} 2

- curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep metadata

akvorado_inlet_metadata_cache_expired_entries_total 0
akvorado_inlet_metadata_cache_hits_total 0
akvorado_inlet_metadata_cache_misses_total 0
akvorado_inlet_metadata_cache_refresh_runs_total 169
akvorado_inlet_metadata_cache_refreshs 0
akvorado_inlet_metadata_cache_size_entries 0
akvorado_inlet_metadata_provider_batched_requests_total 0
akvorado_inlet_metadata_provider_snmp_poller_pending_requests 0

And there is no outputs for these commands: 

- curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep _error
- curl -s http://127.0.0.1:8080/api/v0/inlet/flows\?limit=1

Classifier:
  exporter-classifiers:
    - ClassifySiteRegex(Exporter.Name, "^[^-]+-([^-]+)-", "$1")
    - ClassifyRegionRegex(Exporter.Name, "^([^-])+-[^-]+-", "$1")
    - ClassifyTenant("AS2613")
    - ClassifyRole("br")
  interface-classifiers:
    - |
      ClassifyConnectivityRegex(Interface.Description, "^(?i)(transit|peering):? ", "$1") &&
      ClassifyProviderRegex(Interface.Description, "^\\S+?\\s(?:to )?(\\S+)", "$1") &&
      ClassifyExternal()
    - ClassifyConnectivityRegex(Interface.Description, "^(?i)(cust|core):? ", "$1") &&
      ClassifyInternal()
    - ClassifyInternal()

**Comment by vincentbernat** [comment, marked as answer]
From
```
akvorado_inlet_flow_decoder_flows_total{name="netflow"} 2045
akvorado_inlet_flow_decoder_netflow_flows_total{exporter="192.168.6.90",version="9"} 2045
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="192.168.6.90",type="OptionsDataFlowSet",version="9"} 2012
akvorado_inlet_flow_decoder_netflow_flowset_records_sum{exporter="192.168.6.90",type="OptionsTemplateFlowSet",version="9"} 2012
akvorado_inlet_flow_decoder_netflow_templates_total{exporter="192.168.6.90",obs_domain_id="6",template_id="258",type="options_template",version="9"} 2012
```
You seem to only receiving "options" flow. You should have some TemplateFlowSet and DataFlowSet. Maybe you can check with tcpdump and decode the flows with Wireshark, but it would mean your Cisco is not sending any data, like if you didn't enable sampling on interfaces.

**Reply by khusleek** [reply]
Much appreciated. As you said, it was a misconfiguration on my router. It's working fine now.

**Comment by rein392** [comment]
it worked with vrf on the cisco? ((
